### PR TITLE
streamline-data-prep-file

### PR DESCRIPTION
**Pull request by mwong70-stat660**

### CODE REVIEW

File: STAT697-01_s20-team-2_data_preparation.sas [hunk 1]
```diff
@@ -175,39 +174,33 @@ https://github.com/yxie18-stat697/team-2_project_repo/blob/master/data/enr16.xls
...
         select
-            COUNTY
+             COUNTY
             ,DISTRICT
             ,SCHOOL
-            ,count(*) as row_count_for_unique_id_value
-        from
-            gradaf17
+            ,count(*) 
+            as row_count_for_unique_id_value
```
Comment (yxie18-stat697) [inline STAT697-01_s20-team-2_data_preparation.sas:190]: Yes, this variable is the correct format. snake_case

Comment (mwong70-stat660) [reply STAT697-01_s20-team-2_data_preparation.sas:190]: Thanks for the feedback. It's very helpful for me to visualize what you meant.

File: STAT697-01_s20-team-2_data_preparation.sas [hunk 2]
```diff
@@ -429,9 +403,9 @@ enrollment per county, and the average value of Estimated FTE per county. */
 proc sql;
```
Comment (yxie18-stat697) [inline STAT697-01_s20-team-2_data_preparation.sas:400]: Starting from this line of query, I was trying to combine three datasets (gradaf17, enr16, Staffassign16)  into one table.

File: STAT697-01_s20-team-2_data_preparation.sas [hunk 3]
```diff
@@ -264,39 +248,30 @@ proc sql;
 quit;
 
 
-/* check gradaf17 for bad unique id values, where the column CDS_CODE is 
-intended to be a primary key */
+/* Join gradaf16 and gradaf17 where COUNTY is the primary key. Check for
```
Comment (mwong70-stat660) [inline STAT697-01_s20-team-2_data_preparation.sas]: Here, I tried to join

Comment (mwong70-stat660) [reply STAT697-01_s20-team-2_data_preparation.sas]: @yxie18-stat697

**Pull request merged**